keyboards/qwertykeys/qk80/ansi_625u/readme.md

```diff
@@ -0,0 +1,21 @@
+# qk80 - 625U ANSI PCB
+
+![qk80](https://i.imgur.com/qSXHoWyh.jpg)
```

Comment:
Could this image be changed to match keyboard - current iimage is a tsangan/7u spacebar layout.